src/lib/core/components/computations/StartPage.tsx

```diff
@@ -18,22 +20,35 @@ export function StartPage(props: Props) {
         variables.
       </p>
       <div style={{ display: 'flex', flexDirection: 'column', width: '50%' }}>
-        {apps.map((app) => (
-          <Link
-            to={`${baseUrl}/new/${app.name}`}
-            style={{
-              padding: '1em',
-              border: '1px solid',
-              borderRadius: '.5em',
-              margin: '1em 0',
-            }}
-          >
-            <div style={{ fontWeight: 'bold', fontSize: '1.2em' }}>
-              {app.displayName}
+        {/* orderBy renders available apps ahead of those in development */}
+        {orderBy(apps, [(app) => (plugins[app.name] ? 1 : 0)], ['desc']).map(
+          (app) => (
+            <div
```

Comment:
Is there a reason it's beneficial to have the outer element be a div instead of a Link? Noting that the hierarchy is swapped.

We'll eventually change these buttons to real buttons somehow (that's on me to figure out still!) so for now I think it's good either way, just curious for my own education.

Reply:
Great question! I altered the hierarchy in order to use `cursor: not-allowed` when the link is disabled via `pointer-events: none`. Setting pointerEvents to none essentially disables the mouse event and thus the `cursor: not-allowed` wasn't showing properly. 

[But in re-reading the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events#values), it looks like I could explicitly set the children's `pointer-events` property to auto and maintain the previous hierarchy? We'd need to make a few tweaks to allow the `divs` to occupy the full space of the `Link` such that hovering over any part of `Link` will show the disabled hover effect.

Reply:
Ahhh interesting! Thank you for the explanation!

I think it's okay to leave as is. Since we're going to redo the button styling in a few weeks anyways, it's probably not worth the time now getting them to work a certain way when we may change it/use a different component/etc.